eradiate/scenes/measure/_distantsensorarray.py

```diff
@@ -0,0 +1,218 @@
+from __future__ import annotations
+
+from typing import Dict, List, Tuple
+
+import attr
+import numpy as np
+import xarray as xr
+
+from ._core import SensorInfo, measure_factory
+from ._distant import DistantMeasure
+from ..illumination import DirectionalIllumination
+from ...attrs import documented, parse_docs
+from ...units import symbol
+from ...units import unit_context_kernel as uck
+
+
+@measure_factory.register(type_id="distantarray")
+@parse_docs
+@attr.s
+class DistantArrayMeasure(DistantMeasure):
+    """
+    Distant sensor array measure scene element [``distantarray``].
+
+    This measure scene element is a thin wrapper around the ``DistantSensor``
+    measures. It records either the radiance or reflectance along the directions given by the
+    ``directions`` parameter.
+    """
+
+    directions: np.ndarray = documented(
+        attr.ib(
+            default=np.array([[0.0, 0.0, 1.0]]),
+            converter=np.array,
+        ),
+        doc="A sequence of 3-vectors specifying distant sensor directions.",
+        type="array-like",
+        default="[[0, 0, 1]]",
+    )
+
+    @directions.validator
+    def _directions_validator(self, attribute, value):
+        if value.shape[1] != 3:
+            raise ValueError(
+                f"While initializing {attribute}: "
+                f"Expected shape (N, 3), got {value.shape}"
+            )
+
+    def sensor_infos(self) -> List[SensorInfo]:
+        """
+        Return a tuple of sensor information data structures.
+
+        Returns → list[:class:`.SensorInfo`]:
+            List of sensor information data structures.
+        """
```

Comment:
No need to repeat the docstring if it is inherited. However, it would be nice if the parent method  would document the rationale underlying sensor ID definition.